### PR TITLE
Refactor code to reduce cognitive complexity from 16 to 15 in src/user/settings.js

### PR DESCRIPTION
**Pull request by SarimFaraz**
The purpose of this task is to reduce the cognitive complexity of the file src/user/settings.js

I broke the code down and made two new functions to reduce the cognitive complexity:

validatePaginationValue: This function checks if a given pagination value is within the acceptable range. If the value is not valid, it throws an error.

validateLanguage: This function checks if a given language code is included in the list of valid language codes. If the language code is not valid, it throws an error.

This makes the code easier to read, follow, and fixes the issues.

### CODE REVIEW

**Comment by FatimaHad** [comment]
The code is working well.

**Review by FatimaHad — COMMENTED** [review]
The code looks good, and working well.

**Review by mrahmed2026 — APPROVED** [review]
I approve, the code looks good, passes all the elint tests and was written well!

File: src/user/settings.js [hunk 1]
```diff
@@ -96,32 +95,34 @@ module.exports = function (User) {
...
+
+	// Co-pilot assisted code
+	function validateLanguage(value, languageCodes) {
+		console.log('language is saved');
+		if (value && !languageCodes.includes(value)) {
+			throw new Error('[[error:invalid-language]]');
+		}
+	}
+
+	// Co-pilot assisted code
```
Comment (safa-amin) [inline src/user/settings.js:114]: function looks good and easy to read. the comments are also helpful to better understand the code.

**Review by safa-amin — COMMENTED** [review]
code is functional and well written.

**Pull request merged**